src/code_8001F980.c

```diff
@@ -1185,7 +1191,7 @@ void func_80023BF0(Player *player, s8 arg1, s8 arg2, s8 arg3) {
     }
 }
 
-void func_80023C84(Player *player, s8 arg1, s8 arg2) {
+void shadow_player_render(Player *player, s8 arg1, s8 arg2) {
```

Comment:
render_player_shadow

Now, I know this is going to seem backwards from everything I've talked about. It's just this one needs to have adjective, subject, noun, to feel grammatically correct.